packages/components/src/progress_bar/utils.jsx

```diff
@@ -0,0 +1,9 @@
+export const getPercentage = (input) => {
+  let percent;
+  if (typeof input === 'number' && input < 1) percent = String(input * 100);
```

Comment:
Maybe to simplify it a bit?

```suggestion
  if (typeof input === 'number' && input < 1) return `${input}%`;
  if (input.includes('%')) return String(input);
  return `${input}%`;
```